src/styles/variables.css

```diff
@@ -114,11 +114,12 @@
   --spacing-medium: 0.5px;
 
   /**
-   * @NOTE Weight medium is olny available on the techie font family
+   * @NOTE Weight medium and semi bold is only available on the techie font family
    */
   --family-monospace: "Source Code Pro", "Monaco", "Inconsolata", monospace;
   --family-techie: "IBM Plex Mono", sans-serif;
   --weight-medium: 500;
+  --weight-semi-bold: 600;
```

Comment:
I really don't feel good about adding a new font file _(which we have to distribute)_ for a font-weight that is so minor in size from the previous one....

Reply:
Let's remove it for now, right now it's just 1 place where it's used